README.md

```diff
@@ -1,5 +1,16 @@
 # KubeCF
 
+| Build step | State |
+| ---     | ---   |
+| Linting | [![lint](https://kubecf-github-status.herokuapp.com/state?context=lint)](https://kubecf-github-status.herokuapp.com/url?context=lint) |
+| Building | [![build](https://kubecf-github-status.herokuapp.com/state?context=build)](https://kubecf-github-status.herokuapp.com/url?context=build) |
+| Smoke tests on Diego | [![smoke-diego](https://kubecf-github-status.herokuapp.com/state?context=smoke-diego)](https://kubecf-github-status.herokuapp.com/url?context=smoke-diego) |
+| Smoke tests on Eirini | [![smoke-eirini](https://kubecf-github-status.herokuapp.com/state?context=smoke-eirini)](https://kubecf-github-status.herokuapp.com/url?context=smoke-eirini) |
+| Acceptance Tests on Diego | [![acceptance-diego](https://kubecf-github-status.herokuapp.com/state?context=acceptance-diego)](https://kubecf-github-status.herokuapp.com/url?context=acceptance-diego) |
+| Acceptance Tests on Eirini | [![acceptance-diego](https://kubecf-github-status.herokuapp.com/state?context=acceptance-eirini)](https://kubecf-github-status.herokuapp.com/url?context=acceptance-eirini) |
+
+_[Instructions on how to setup these badges](https://github.com/jimmykarily/github_branch_status)_
```

Comment:
It looks good for me! the only thing is that it might have sense to group those setup in some page, because it might not fit on the front readme (as the majority won't be interested on how to re-deploy the app, but we do)

Reply:
I couldn't find a more appropriate place and I didn't want to create a page just for this. I was hoping the linked project has all the needed information so a simple link to that would be enough here. Do you think we need additional instructions? Any idea on where to put them?

Reply:
> I couldn't find a more appropriate place and I didn't want to create a page just for this. 

Me neither.. that's why I don't think we should block this PR for this, just a minor nitpick that we could address in the future

I think the README in the project it's clear enough, and no more instructions are needed :+1:  just not sure where it would be best to put the reference to it, maybe makes sense to have a `.md` under `docs/` that explains all the infra bits needed (it is maybe also a good fit for the concourse pipeline(s) infrastructure)

Reply:
When we have enough things in the README that we need to move to other places, maybe a structure for docs will become more obvious :). Let's merge it and decide later.